automation/test.sh

```diff
@@ -47,12 +47,14 @@ elif [[ $TARGET =~ cnao ]]; then
   export KUBEVIRT_DEPLOY_CDI=false
 elif [[ $TARGET =~ sig-network ]]; then
   export KUBEVIRT_WITH_CNAO=true
-  export KUBEVIRT_PROVIDER=${TARGET/-sig-network/}
-  export KUBEVIRT_DEPLOY_ISTIO=true
   export KUBEVIRT_DEPLOY_CDI=false
-  if [[ $TARGET =~ k8s-1\.1.* ]]; then
+  # FIXME: https://github.com/kubevirt/kubevirt/issues/9158
+  if [[ $TARGET =~ no-istio ]]; then
     export KUBEVIRT_DEPLOY_ISTIO=false
+  else
+    export KUBEVIRT_DEPLOY_ISTIO=true
```

Comment:
what about moving the responsibility of setting KUBEVIRT_DEPLOY_ISTIO to the infra ?
here we will just use the value on line 369

Reply:
Using kubevirtci env variables as a way to communicate from project-infra to kubevirt which tests should be executed sounds odd to me. I don't see a big benefit of that over the current implementation. I would prefer to leave things as they are now